jokes/index.json

```diff
@@ -448,5 +448,29 @@
     "type": "general",
     "setup": "What do you give to a lemon in need?",
     "punchline": "Lemonaid."
+  },
+  {
+  "id": 76,
+  "type": "general",
+  "setup": "I used to think the brain was the most important organ",
+  "punchline": "Then I thought, look what’s telling me that."
+  },
+  {
+  "id": 77,
```

Comment:
Can you indent these like the other jokes in the file?

Reply:
see the new commit